protocol/bc/merkle_test.go

```diff
@@ -1,18 +1,18 @@
-package validation
+package bc_test
```

Comment:
It looks like the uses of package vm here are just for some constants that have no relevance to the meaning of the tests. Seems like it would be pretty trivial to remove that dependency. In fact I just opened#799 to do that. It would be preferable imo to land #799 first, then do this pr without having to resort to the bc_test trick.